pkg/quarterdeck/config/config.go

```diff
@@ -32,6 +32,7 @@ type Config struct {
 	Database      DatabaseConfig
 	Token         TokenConfig
 	Sentry        sentry.Config
+	RateLimit     RatelimitConfig
```

Comment:
```suggestion
	RateLimit     RatelimitConfig `split_words:"true"`
```

☝🏽 will change the environment variable from `$QUARTERDECK_RATELIMIT_BURST` to `$QUARTERDECK_RATE_LIMIT_BURST`.